dll/win32/shell32/CCopyMoveToMenu.cpp

```diff
@@ -184,60 +188,44 @@ CCopyMoveToMenu::DoRealFileOp(LPCMINVOKECOMMANDINFO lpici, LPCITEMIDLIST pidl)
 }
 
 static HRESULT
-DoGetFileTitle(CStringW& strTitle, IDataObject *pDataObject)
+DoGetFileTitle(const CIDA *pCIDA, CStringW& strTitle, IDataObject *pDataObject)
```

Comment:
What then is the `pDataObject` parameter used for?
(In the callers of this function, you now create the pCIDA from the pDataObject, so it seems the former is what now replaces the latter.)

Reply:
Good point, I'll remove it.